content/_data/figures.yaml

```diff
@@ -1449,13 +1450,15 @@ figure_list:
     label: "Figure 27.2"
     caption: "SEM images of the modes of deposition for the consolidation treatments. Presence of interfibrillar bridges is indicated by red markings. (w) water; (h) heptane."
     credit: "Image: Alexandra Bridarolli and Oleksandr Nechyporchuk"
+    alt: "Different levels of magnification of the untreated canvas and each treatment. Red markings are present in Mc+CNC (w) and Sil-PEI-CMC on enlarged magnification."
```

Comment:
I don't think we need to belabor it in this book, but in the context of your work with Nola on our main alt text guidelines, I wonder about acronyms/abbreviations like `Mc+CNC (w) and Sil-PEI-CMC` and `(*E’*)` in alt text. Should they instead be written out in plain language? Do they have meaning as is to the audience? I'm not sure.